core/src/test/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClientTest.java

```diff
@@ -132,6 +135,7 @@ private Properties createProducerProps() {
     props.put("key.serializer.class", "kafka.serializer.StringEncoder");
     props.put("metadata.broker.list", brokerList);
     props.put(SCHEMA_REGISTRY_URL, restApp.restConnect);
+    props.put(KafkaAvroSerializerConfig.ENABLE_AUTO_SCHEMA_REGISTRATION_CONFIG, true);
```

Comment:
Somewhat unintuitive, but use "true" instead of true

This has to do with how the old producer (as opposed to "newProducer") parses configs. This is most likely the cause of your test error below.